src/main/java/io/sphere/sdk/meta/CustomObjectDocumentation.java

```diff
@@ -148,6 +150,12 @@ Instead of pure JSON it is encouraged to use Java objects (POJO) with the Jackso
 
  {@include.example io.sphere.sdk.customobjects.migrations.CustomObjectsMigrationsTest#exampleForMigrationCall()}
 
+
+ <h3 id="traps">Traps</h3>
+
+ <p>Unlike other query models, {@link CustomObjectQuery#model()} takes a type parameter of the result type of the custom object.</p>
```

Comment:
I would continue the sentence with something that focuses the attention on the type parameter declaration, to make sure they understand what we mean with that. Something like "Notice that it is necessary to explicitly declare the type when requesting the query model, as shown in the following examples" (better phrased).